docs/lang/articles/type_system/type.md

```diff
@@ -9,17 +9,17 @@ Taichi is a statically typed programming language, meaning that the type of a va
 
 Let's see a quick example:
 
-```python {3-5}
+```python skip-ci:NotRunnable
 @ti.kernel
 def test():
     x = 1  # x is the integer 1
     x = 3.14  # x is an integer, so the value 3.14 is cast to 3 and x takes the value 3
     x = ti.Vector([1, 1])  # Error!
 ```
 
-- Line 3: `x` is an integer because it is assigned an integer value the first time it is declared.
-- Line 4: `x` is reassigned a floating-point number 3.14 but takes the value 3. This is because 3.14 is automatically cast to integer 3 to match the type of `x`.
-- Line 5:  The system throws an error, because `ti.Vector` cannot be cast into an integer.
+- Line 3: `x` is an integer since it is assigned an integer value when it is declared for the first time.
+- Line 4: `x` is reassigned a floating-point number 3.14. However, the value of `x` is 3 instead of 3.14. This is because 3.14 is automatically cast to an integer 3 to match the type of `x`.
+- Line 5: In this line, we are assigning a `ti.Vector` to `x`. However, the system throws an error because a `ti.Vector` cannot be cast into an integer.
```

Comment:
Hmm why do we change this part? IMO the original sentences are OK :-)

Reply:
It's polished by chatgpt so that more like a tech doc style ~